src/index.css

```diff
@@ -54,4 +54,75 @@ h4 {
 
 p {
   margin-block: 0.5em;
+}
+
+img {
+  width: 80%;
+}
+
+figure {
+  margin: 0 0 2em 0;
+}
+
+figcaption {
+  text-align: right;
+  font-size: 0.9em;
```

Comment:
Je gebruikt soms em en soms px. Zorg ervoor dat je consistent bent in jouw keuze!